### DISCUSSION TITLE
Support for php.new (herd-lite)

### DISCUSSION THREAD

**Discussion by brunogoossens** [php/pie#297] [Ideas]
It would be nice if this packages works out of the box with https://php.new (herd-lite).

**Comment by asgrim** [comment]
I'm unfamiliar with this tool. What do you imagine PIE doing to support this? Can you give some examples please?

**Comment by alexandre-daubois** [comment]
It seems to be a tool installing PHP, Composer and Laravel by only executing one command in your terminal. It would be neat indeed if PIE was included! But it doesn't seem open-source so not sure how to approach this... 🤔

**Comment by brunogoossens** [comment]
There is a community repo: https://github.com/beyondcode/herd-community

I also added a comment there: https://github.com/beyondcode/herd-community/discussions/184#discussioncomment-11446646

It would be nice if the 2 projects work together to have a great PHP install experience.

**Comment by kynx** [comment]
Woah! That’s one heck of a long thread from people bleating “compile this for me”. 

There is no code in the linked repro and I didn’t go looking, but maybe they could build on top of PIE? Does php.new make compiling stuff yourself that difficult?

Not sure what to say to folks who don’t know what they’ve just installed though. Wake up?

🙂

**Comment by brunogoossens** [comment]
Why are you thinking they don't know what they installed?
Here is the bash script: https://php.new/install/linux. You need php-dev to compile. That's not included. That's why PIE does not work. The solution is probably at their side, but it would be nice if the projects work together to make it happen. A one-liner to get started as a new developer is super convenient. Afterwords, the ability to extending it with PIE extensions would be nice.
